doc/glossary.rst

```diff
@@ -810,7 +810,17 @@ Class APIs and Estimator Types
         * :term:`predict` if :term:`inductive`
 
     density estimator
-        TODO
+        An :term:`unsupervised` estimation of input density without a labeled response.
+        Most commonly used techniques are `Histograms <https://scikit-learn.org/stable/modules/density.html#density-estimation-histograms>`_,
+        `GaussianMixture`,
+        and `KernelDensity` estimation.
+
+        * `Histograms <https://scikit-learn.org/stable/modules/density.html#density-estimation-histograms>`_
+          visually represents the density of specific bins.
+        * Gaussian Mixtures are discussed in `Clustering`.
```

Comment:
Here `Clustering` is not a link. If you want to link to the glossary term, use 
```py
:term:`clusterer`
```
If you want to link to the clustering page of the doc, use:
```py
:ref:`clustering`
```